opentelemetry-api/opentelemetry/trace/__init__.py

```diff
@@ -115,40 +124,46 @@ def start_span(self, name: str, parent: 'Span') -> Iterator['Span']:
 
         is equivalent to::
 
-            span = tracer.create_span(name, parent=tracer.get_current_span())
+            span = tracer.create_span(name)
             with tracer.use_span(span):
                 do_work()
 
         Args:
             name: The name of the span to be created.
-            parent: The span's parent.
+            parent: The span's parent. Defaults to the current span.
 
         Yields:
             The newly-created span.
         """
 
-    def create_span(self, name: str, parent: 'Span') -> 'Span':
-        """Creates a new child span of the given parent.
+    def create_span(self,
+                    name: str,
+                    parent: 'Span' = CURRENT_SPAN) -> 'Span':
```

Comment:
How to create a span with a SpanContext as parent?

https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/tracing-api.md#span-creation

Reply:
Here goes the original proposal https://github.com/open-telemetry/opentelemetry-specification/pull/136#discussion_r295010329

Reply:
Good point. The type of `parent` should probably be not `Span` but either `SpanContext` or (I'd prefer this:) `typing.Union[Span, SpanContext]`.

Reply:
Oh, thanks, I knew I had forgotten something ;) 

So the options here are:

1. Use parent for **both** values (and use the `typing.Union` detail).
2. Use separated parameters and do not have this problem (in this case, we would need to give precedence to one of them).

My impression is that 1) would work just fine. Thoughts?

Reply:
Agree, I would go with 1), too.